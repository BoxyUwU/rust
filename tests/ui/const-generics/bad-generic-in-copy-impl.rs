#[derive(Copy, Clone)]
pub struct Foo {
    x: [u8; SIZE],
    //~^ ERROR the constant `1` is not of type `usize`
}

const SIZE: u32 = 1;

fn main() {}
