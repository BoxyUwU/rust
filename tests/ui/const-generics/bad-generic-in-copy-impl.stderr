error: the constant `1` is not of type `usize`
  --> $DIR/bad-generic-in-copy-impl.rs:3:8
   |
LL |     x: [u8; SIZE],
   |        ^^^^^^^^^^ expected `usize`, found `u32`

error: aborting due to previous error

